transactions/stakingCollection/register_node_old.cdc

```diff
@@ -0,0 +1,52 @@
+import Crypto
+import FlowStakingCollection from "FlowStakingCollection"
+
+/// Registers a delegator in the staking collection resource
+/// for the specified node information and the amount of tokens to commit
+
+transaction(id: String,
+            role: UInt8,
+            networkingAddress: String,
+            networkingKey: String,
+            stakingKey: String,
+            amount: UFix64,
+            machineAccountKey: String, 
+            machineAccountKeySignatureAlgorithm: UInt8, 
+            machineAccountKeyHashAlgorithm: UInt8) {
+
+    let stakingCollectionRef: auth(FlowStakingCollection.CollectionOwner) &FlowStakingCollection.StakingCollection
+
+    prepare(account: auth(BorrowValue) &Account) {
+        self.stakingCollectionRef = account.storage.borrow<auth(FlowStakingCollection.CollectionOwner) &FlowStakingCollection.StakingCollection>(from: FlowStakingCollection.StakingCollectionStoragePath)
+            ?? panic(FlowStakingCollection.getCollectionMissingError(nil))
+
+        if let machineAccount = self.stakingCollectionRef.registerNode(
```

Comment:
doesn't `registerNode` expect the PoP parameter? I guess we need `registerNodeOld` no?

Reply:
That makes sense, thanks for clarifying. This old transaction would break after the contract upgrade which is expected